src/components/auth/SocialMediaLogin.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import QueryString from 'query-string';
+import { socialAuth } from '../../actions/index';
+
+export class SocialMediaLogin extends Component {
+  componentDidMount() {
+    // eslint-disable-next-line react/prop-types
```

Comment:
Why though?

This component should have propType validations

Reply:
Can you explain more on what mean by the component being a propType validations?

Reply:
@okoroemeka here's a link to a refresher: https://reactjs.org/docs/typechecking-with-proptypes.html